yt_dlp/extractor/medaltv.py

```diff
@@ -81,6 +79,9 @@ class MedalTVIE(InfoExtractor):
     def _real_extract(self, url):
         video_id = self._match_id(url)
 
+        if '?mobilebypass=true' not in url:
+            url += ('&' if '?' in url else '?') + 'mobilebypass=true'
+
         webpage = self._download_webpage(url, video_id)
```

Comment:
```suggestion
        webpage = self._download_webpage(update_url_query(url, {'mobilebypass': 'true'}), video_id)
```
Import `update_url_query` from `..utils`.